src/components/InputFile/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+import './InputFile.css'
+
+const InputFile = ({ onChangeHandler }) => {
+
+      return (
+            <div className="upload-area__drop">
+                <input
+                    className="upload-area__drop__input"
+                    type="file"
+                    name="file"
+                    onChange={onChangeHandler}
+                />
+                <img
+                    className="upload-area__drop__icon"
+                    src="https://www.shareicon.net/data/256x256/2015/09/05/96087_cloud_512x512.png"
```

Comment:
I think it would be best to load this locally but that's fine for now I guess. :stuck_out_tongue:

Reply:
True, I'll update that.